deploy/cephfs/kubernetes/v1.14+/helm/templates/nodeplugin-daemonset.yaml

```diff
@@ -108,6 +108,28 @@ spec:
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
           resources:
+{{ toYaml .Values.nodeplugin.plugin.resources | indent 12 }}
```

Comment:
can you update provisioner helm chat for the same? for both rbd and cephfs

Reply:
Good catch! Missed it being added there :)